backend/src/test/resources/application-test.yml

```diff
@@ -41,6 +41,7 @@ registry:
   external-subject-id-wildcard-prefix: "PUBLIC_READABLE"
   external-subject-id-wildcard-allowed-types: manufacturerPartId,assetLifecyclePhase
   idm:
+    identity-provider: keycloak
```

Comment:
Should this be configurable in the charts as well?

Reply:
Thanks for the question! I meant, that we might want to allow the end users to configure this property when they are starting their application using Kubernetes. This project is providing the relevant Helm charts under this location: https://github.com/eclipse-tractusx/sldt-digital-twin-registry/tree/main/charts/registry

In my opinion, it would be valuable to allow this as otherwise it would be harder to use the new code. Also, we could set "keycloak" as default value for the production configuration as well to ensure backward compatibility in case someone does not update their configuration.

Reply:
Thank you for the info. We do not use Helm Charts so I did not change the code here. Setting keycloak as a default would be suitable to gurantee barckward compatibility

Reply:
Thank you for the additional context! Yes, it is easy to overlook this, but having it configurable in the Helm charts can help others who will use the feature later. Could you please make sure to change that part as well?

Reply:
Hi @istvan-nagy-epam I'm absolutely no expert in Helm charts. What exactly do you expect here? Make it configurable in the Helm charts if keycloak or cognito should be used?